### DISCUSSION TITLE
How did you make the Google work

### DISCUSSION THREAD

**Discussion by YourMomUgly** [UseInterstellar/Interstellar#488] [General]
I have my own unblocker that i made and i want to put google on it but i cant find out how too can you help me?

**Comment by 0mhx** [comment]
nah

**Comment by YourMomUgly** [comment]
Secret? 😂

**Reply by YourMomUgly** [reply]
But thanks for reaching out to me

**Comment by MadjikDotPng** [comment]
What do you even mean?
